automation/Dockerfile_balena-push-env

```diff
@@ -31,5 +31,6 @@ RUN curl -sSL https://github.com/balena-io/balena-cli/releases/download/v$BALENA
   rm -rf balena-cli.zip balena-cli
 
 COPY manage-docker.sh /
-COPY balena-push-os-version.sh /
+# Do not fail if balena-hostapp-extension.sh not present for backwards compatibility
+COPY balena-push-os-version.sh balena-hostapp-extension.s[h] /
```

Comment:
how does the `s[h]` thing work, is it related to the comment above about making it backwards compatible?

Reply:
hi @nazrhom yes, the `COPY` command does a conditional copy if you use a wildcard - so here I use `[h]` so that only a whole name match applies. This is backwards compatible because if `balena-hostapp-extension.sh` does not exist, as will happen once this is merged when building older releases, the COPY will not fail.